framework/filters/PageCache.php

```diff
@@ -123,6 +124,13 @@ class PageCache extends ActionFilter
      * @since 2.0.4
      */
     public $cacheHeaders = true;
+    /**
+     * @var array a list of placeholders for embedding dynamic contents. This property
+     * is used internally to implement the content caching feature. Do not modify it.
+     * @internal
```

Comment:
It's a [convention](https://github.com/yiisoft/yii2/blob/master/framework/base/View.php#L363) I have to follow.